src/start/RealmGate.js

```diff
@@ -0,0 +1,44 @@
+/* @flow */
+import React, { PureComponent } from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import connectWithActions from '../connectWithActions';
+import type { Actions } from '../types';
+import { Screen, Centerer, ZulipButton } from '../common';
+
+const componentStyles = StyleSheet.create({
+  divider: {
+    height: 20,
+  },
+});
+
+type Props = {
+  actions: Actions,
+};
+
+class RealmGate extends PureComponent<Props> {
+  props: Props;
+
+  navToRealmScreen = () => {
+    const { actions } = this.props;
+    actions.navigateToAddNewAccount();
```

Comment:
looking [here](https://github.com/zulip/zulip-mobile/blob/master/src/nav/navActions.js#L43) I guess `navigateToAddNewAccount` expects a string argument which specifies the realm.

Reply:
Agreed.

I highly recommend (for everyone working on zulip-mobile) getting Flow integrated into your workflow; it will help you spot this kind of thing sooner. E.g., @agzuniverse, try running `yarn flow` -- it prints out a nice error for this.

Reply:
Ah ok. I've changed it to an empty string, since there appears to be no need to pass a realm as an argument here.